packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -1,3 +1,34 @@
+<!-- prettier-ignore -->
+<% [ 
```

Comment:
Why do we need to put the json inside the HTML example? It would be simpler to just use the text in the example

Reply:
@nichia

Reply:
I went the path of using ejs to render the same html once for English and then Spanish to make the html example shorter. Do you think it'll be clearer to just write the full same html code twice with it's respective language?

Reply:
It does look clearer. Will update